R/utils.R

```diff
@@ -157,8 +157,9 @@ render_dependencies <- function(dependencies, local = TRUE, prefix=NULL) {
     if (!is.null(dep$package)) {
       # the gsub line is to remove stray duplicate slashes, to
       # permit exact string matching on pathnames
-      dep_path <- file.path(dep$src$file,
-                            dep$script)
+      dep_path <- ifelse(!is.null(dep$script),
+                         file.path(dep$src$file, dep$script),
+                         file.path(dep$src$file, dep$stylesheet))
```

Comment:
🌴 
https://github.com/plotly/dashR/pull/94/files#diff-cbb7235afee10c3646161358a7655b0aR465

Reply:
🐪 fixed in d9d66c13062da253b0f77bb131126f64f12e8516